public/js/gulliver-config.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2015-2016, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-env browser */
+
+export default class Config {
+  constructor(element) {
+    if (!element) {
```

Comment:
Maybe it's better to take the CSS selector here. That way if it doesn't exist the error message can report the actual selector that failed. Similarly, if the `JSON.parse(...)` fails, can report the error against the selector, instead of blowing up in an apparently unrelated location.